cmd/bankapp/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/iliyanmotovski/bankv2/bank/api"
+	"github.com/iliyanmotovski/bankv2/bank/domain"
+	"github.com/iliyanmotovski/bankv2/bank/persistence"
+	"github.com/justinas/alice"
+	"gopkg.in/mgo.v2"
+	"net/http"
+	"time"
+)
+
+func main() {
+	session, err := mgo.DialWithTimeout("localhost:27016", time.Second*10)
+	if err != nil {
+		fmt.Println(err)
+	}
+	defer session.Close()
+	session.SetMode(mgo.Monotonic, true)
+
+	renew := domain.Renew{RenewSessionCounter: make(chan bool)}
+
+	r := mux.NewRouter()
+
+	mongoSessionRepo := persistence.NewSessionRepository(*session, "bank")
```

Comment:
Code uses sessionStore but here it's called: sessionsRepository

I don't mind which one to be, but only one naming convention need to be used. 

Name of variable should be just: sessionStore or sessionRepository as If I change the implementation of the store to: memory.NewInMemorySessionStore() I don't have to change on many places in my code.